src/troute-routing/troute/routing/compute.py

```diff
@@ -218,38 +218,41 @@ def _prep_reservoir_da_dataframes(reservoir_usgs_df, reservoir_usgs_param_df, re
     return reservoir_usgs_df_sub, reservoir_usgs_df_time, reservoir_usgs_update_time, reservoir_usgs_prev_persisted_flow, reservoir_usgs_persistence_update_time, reservoir_usgs_persistence_index,  reservoir_usace_df_sub, reservoir_usace_df_time, reservoir_usace_update_time, reservoir_usace_prev_persisted_flow, reservoir_usace_persistence_update_time, reservoir_usace_persistence_index, waterbody_types_df_sub
 
 def compute_nhd_routing_v02(
-    connections,
-    rconn,
-    wbody_conn,
-    reaches_bytw,
+    network,
```

Comment:
There is a case to be made about keeping this call the same as before and expanding the args from the network at the calling code.  It might be worth discussing the merits and/or a possible proxy to maintain the `compute_nhd_routing_v02` backwards compatibility and possibly deprecate it in the future.

Reply:
Happy to discuss this further. Calling the main routing function has a couple levels and I can see the utility in keeping the bottom levels the same to maintain backwards compatibility. But I do think minimizing these inputs makes the code much more readable and would help with future maintenance.